packages/harpparts/src/packages/slice-matrix/slice-matrix.ts

```diff
@@ -0,0 +1,10 @@
+type Row<T> = ReadonlyArray<T>
```

Comment:
This is a duplicate of a package in harpguru-core. It's time this package found it's way up a level at this point.